app/views/bars/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>All Bars</h1>
+
+  <ul><% @bars.each do |bar| %>
```

Comment:
Your `each` loop should be on a new line from your `ul`, and make sure it gets indented properly when you change that!